client/reader/following/main.jsx

```diff
@@ -44,6 +45,9 @@ const FollowingStream = props => {
 		<Stream { ...props }>
 			{ config.isEnabled( 'reader/following-intro' ) && <FollowingIntro /> }
 			<MaterialButton className="following__button">Howdy</MaterialButton>
+			<MaterialTextField label="Dog">
```

Comment:
it's probably more accurate to put `Canine` - I'm concerned about losing semantic value with `Dog`

Reply:
"Howdy Dog" just reads better though.

Reply:
[You're the man now, dog!](
http://yourethemannowdog.ytmnd.com)

Reply:
Better check `canineuse.com` if browsers support this.